content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,193 @@
+
+- **`create-db-worker`**: Provisions the database and schedules deletion.
+- **`claim-db-worker`**: Handles the claim and transfer process, including OAuth2 authentication.
+- **`create-db` CLI**: The entry point for the CLI command.
+
+## 1. Creating a database
+
+Start by running the following command:
+
+```terminal
+npx create-db
+```
+
+This triggers the CLI logic, which:
+
+1. Parses the command-line arguments *(such as region or interactive mode)*.
+2. If interactive mode is enabled, prompts the user to select a region from the available options.
+3. Provisions a temporary Prisma Postgres database in the selected region.
+4. Schedules the database for automatic deletion in 24 hours via a [Cloudflare Workflow](https://developers.cloudflare.com/workflows/).
+5. Displays the connection strings and a claim link for the next step.
+
+:::note Endpoints used
+- Fetches available regions: `GET https://api.prisma.io/v1/regions`
+- Creates a new project: `POST https://api.prisma.io/v1/projects`
+- Deletes a project: `POST https://api.prisma.io/v1/projects/{projectID}/delete`
+:::
+
+#### Request from CLI to create worker
+
+```ts file=create-db
+await fetch(`${CREATE_DB_WORKER_URL}/create`, {
+  method: "POST",
+  headers: { "Content-Type": "application/json" },
+  body: JSON.stringify({ region, name, utm_source: CLI_NAME }),
+});
+```
+
+#### Create worker endpoint
+
+```ts file=create-db-worker
+const prismaResponse = await fetch('https://api.prisma.io/v1/projects', {
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization: `Bearer ${env.INTEGRATION_TOKEN}`,
+  },
+  body: payload,
+});
+```
+
+If successful, the worker returns the new project details, including the `projectID` and connection strings. The worker also triggers a [workflow](https://github.com/prisma/create-db/blob/main/create-db-worker/src/delete-workflow.ts) to schedule deletion of the database in 24 hours.
+
+The CLI displays the connection strings and a claim link:
+
+```txt
+┌  🚀 Creating a Prisma Postgres database
+│
+│  Provisioning a temporary database in us-east-1...
+│
+│  It will be automatically deleted in 24 hours, but you can claim it.
+│
+◇  Database created successfully!
+│
+│
+●  Connect to your database →
+│
+│    Use this connection string optimized for Prisma ORM:
+│
+│    prisma+postgres://accelerate.prisma-data.net/?api_key=<api_key>
+│
+│
+│    Use this connection string for everything else:
+│
+│    postgresql://<user>:<password>@db.prisma.io:5432/postgres
+│
+│
+◆  Claim your database →
+│
+│    Want to keep your database? Claim for free via this link:
+│
+│    https://create-db.prisma.io?projectID=<projectID>&utm_source=create-db&utm_medium=cli
+│
+│    Your database will be deleted on DD/MM/YYYY, HH:MM:SS if not claimed.
+│
+└
+```
+
+The claim link points to the claim worker, passing the `projectID` and UTM parameters.
+
+## 2. Beginning the claim flow
```

Comment:
Same here and the following sections. Explain to the partner what they need to do to let their users claim a db, and not what `create-db` does, that should only be added to bring an example to illustrate the instructions.